### DISCUSSION TITLE
Do you need a profile picture of yourself?

### DISCUSSION THREAD

**Discussion by romanobro56** [community/community#58322] [GitHub Education]
### Hi everyone,

Hello all, 
I am still getting the "update your github user profile" reason as to why my application for the student developer pack is being rejected, except that I have done all the things that it said to do. The only thing that I can think of is that I dont have a profile picture of me. I dont want to put my face on my profile right now but is that the thing that is preventing me from getting the developer pack? Can someone please review my profile and tell me if I'm missing anything? thank you

**Comment by the-officialjosh** [comment]
Yes, these are measures set to confirm that you are who your ID is. Doing this heightens your chances of also being selected

**Comment by mrepol742** [comment]
You dont need a picture of yourself, you just need to use your real name as name in your github account why so? I have github student pack enable and this is always my profile anime.

Tips: Just use school issued university email address and use the latest school documents (must have a current school year like 2022-2023 something) the picture must be clear.

If you are enrolled in a online one and they gave you online certificate unfortunately they will reject you. Because a lot of people are misusing the github edu some of them uses forgery to fake documents.

**Comment by samus-aran** [comment]
Hi @romanobro56 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
